### ISSUE TITLE
[Bug]  (SVDKL)  Shapes are not broadcastable for mul operation

### ISSUE THREAD

**Issue by NeelKanwal** [cornellius-gp/gpytorch#1808]
Hi,

I am trying SVDKL model for multiclass classification. I am trying the example mentioned in NN_intergration which works fine for CIFAR-10.
 
but when I change the data to another custom dataset it again gives the same issue with both versions of DenseNet. Every other setting is pretty the same. I have resized the input images to 224*224.

I saw some similar issues open [#1144](https://github.com/cornellius-gp/gpytorch/issues/1144), I am not able to figure out how to solve it.
 
Did someone find a fix to this? It would be very helpful to hear the insight on this issue. 

Here is the trace. 

File "/home/neel/DKLModel/my_functions.py", line 104, in train
    output = model(data)
  File "/home/neel/miniconda3/envs/DKL/lib/python3.6/site-packages/gpytorch/module.py", line 30, in __call__
    outputs = self.forward(*inputs, **kwargs)
  File "/home/neel/DKLModel/my_functions.py", line 73, in forward
    res = self.gp_layer(features)
  File "/home/neel/miniconda3/envs/DKL/lib/python3.6/site-packages/gpytorch/models/approximate_gp.py", line 81, in __call__
    return self.variational_strategy(inputs, prior=prior, **kwargs)
  File "/home/neel/miniconda3/envs/DKL/lib/python3.6/site-packages/gpytorch/variational/independent_multitask_variational_strategy.py", line 56, in __call__
    function_dist = self.base_variational_strategy(x, prior=prior, **kwargs)
  File "/home/neel/miniconda3/envs/DKL/lib/python3.6/site-packages/gpytorch/variational/_variational_strategy.py", line 129, in __call__
    **kwargs,
  File "/home/neel/miniconda3/envs/DKL/lib/python3.6/site-packages/gpytorch/module.py", line 30, in __call__
    outputs = self.forward(*inputs, **kwargs)
  File "/home/neel/miniconda3/envs/DKL/lib/python3.6/site-packages/gpytorch/variational/grid_interpolation_variational_strategy.py", line 93, in forward
    predictive_mean = left_interp(interp_indices, interp_values, inducing_values.unsqueeze(-1))
  File "/home/neel/miniconda3/envs/DKL/lib/python3.6/site-packages/gpytorch/utils/interpolation.py", line 183, in left_interp
    output_shape = _matmul_broadcast_shape(interp_shape, rhs.shape)
  File "/home/neel/miniconda3/envs/DKL/lib/python3.6/site-packages/gpytorch/utils/broadcasting.py", line 57, in _matmul_broadcast_shape
    bc_shape = _mul_broadcast_shape(batch_shape_a, batch_shape_b)
  File "/home/neel/miniconda3/envs/DKL/lib/python3.6/site-packages/gpytorch/utils/broadcasting.py", line 20, in _mul_broadcast_shape
    raise RuntimeError("Shapes are not broadcastable for mul operation")
RuntimeError: Shapes are not broadcastable for mul operation
Process finished with exit code 137

**Comment by NeelKanwal** [comment]
Hi,

Here is a little update on the issue. I tried to resize my dataset by applying a transform (32,32), very similar to CIFAR 10 and it works fine.

the original size of input images is 512*512 and I want to resize them to 224*224. I am aiming to use the same architecture by replacing Densenet with a couple of different neural networks. 

Is there something I am missing with Densenet inputs in particular? I also tried to modify network size with block_config by holding the size to 224*224.

Thank you.
